LarryKarani/iReporterApi
# #162425359 Test incident model

__What does This PR do?__
This PR creates tests for the incident mode that handles the CRUD operation related to incidents

__Description of Tasks to be completed__
 - test create an incident
- test update an incident location
- test delete an incident
- test update a comment
- test get all incidences

__How should this be manually tested?__
 - clone the repo
- create a virtualenv and activate it
- install all the projects requirements by running `pip install -r requirements.txt`
- run the test by running 'pytest'

__Any background information ?__
The build is failing because the feature has not been fully implemented yet

__Relevant PT stories__
#162425359
